### PR TITLE
operator: update image name and version

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
https://sdk.operatorframework.io/docs/cli/operator-sdk_run_bundle/
According to the documentation of operator-sdk, docker.io should be speficied.

In addition, we update the operator version to 0.25.0.

Also, publishing to operatorhub part in DEVEL.md has been updated/fixed.

Signed-off-by: Hyeongju Johannes Lee <hyeongju.lee@intel.com>

### CODE REVIEW

**Comment by mythi** [comment]
> @hj-johannes-lee @mythi We started to [push to ghcr.io](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/.github/workflows/build-push.yml#L81) recently. would it make sense to stop using docker.io?

these are private (temporary) images. we will continue to push to docker.io

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ GOLANGCI_LINT_VERSION ?= v1.50.0
 KIND_VERSION ?= v0.16.0
 GOLICENSES_VERSION ?= v1.4.0
 # Current Operator version
-OPERATOR_VERSION ?= 0.24.0
+OPERATOR_VERSION ?= 0.25.0
```
Comment (mythi) [inline Makefile:17]: as discussed, see if `TAG` can be reused

Comment (hj-johannes-lee) [reply Makefile]: Yeap, I changed! :)

Comment (mythi) [reply Makefile]: What about the changes needed in `DEVEL.md`

**Review by mythi — CHANGES_REQUESTED** [review]
update `DEVEL.md`

File: DEVEL.md [hunk 2]
```diff
@@ -137,17 +139,19 @@ $ git clone https://github.com/<GitHub Username>/community-operators
 
 Generate bundle and build bundle image:
 ```
-$ make bundle OPERATOR_VERSION=0.X.Y CHANNELS=alpha DEFAULT_CHANNEL=alpha
+$ make bundle CHANNELS=alpha DEFAULT_CHANNEL=alpha
```
Comment (mythi) [inline DEVEL.md:142]: is there a precondition that this is run in a release branch?

Comment (hj-johannes-lee) [reply DEVEL.md:142]: Umm, if we put 'set-version' as the begining of 'make bundle', then I do not think so.

Comment (hj-johannes-lee) [reply DEVEL.md:142]: Ah, actually true.

Then,,,
1. would we put 'TAG' there..?
or
2. would we just keep OPERATOR_VERSION in Makefile, but as OPERATOR_VERSION ?= $TAG?

Comment (hj-johannes-lee) [reply DEVEL.md]: I thought first option would be good enough, but if you have different opinion, let me know.!

**Review by mythi — APPROVED** [review]

**Review by mythi — APPROVED** [review]

**Pull request merged**